README.md

```diff
@@ -10,27 +10,30 @@ Open Source community of contributors: how it works
 ========
 
 ### Shortcuts
-Already contributing to OpenFarm? You're welcome to [introduce yourself]((https://docs.google.com/forms/d/e/1FAIpQLSfsKUABGJSO3gSfDUrdXymxP-GipdTLxbxsmR2FyxGdblN1-w/viewform))!
+Already contributing to OpenFarm? You're welcome to [introduce yourself](https://docs.google.com/forms/d/e/1FAIpQLSfsKUABGJSO3gSfDUrdXymxP-GipdTLxbxsmR2FyxGdblN1-w/viewform)!
+
 Want to dig head first into our todo list of Github tasks? [Make yourself at home](https://github.com/openfarmcc/OpenFarm/projects)!
 
 
 ### About
 
-[OpenFarm](http://openfarm.cc) is a free and open database and web application for farming and gardening knowledge. One might think of it as the Wikipedia for growing plants, though it functions more like a cooking recipes site. The main content are Growing Guides: creative, crowd-sourced, single-author, structured documents that include all of the necessary information for a person or machine to grow a plant, i.e.: seed spacing and depth, watering regimen, recommended soil composition and companion plants, sun/shade requirements, etc. In this Freebase platform, gardeners can find answers to questions like &ldquo;How do I grow tomatoes?&rdquo; 
+[OpenFarm](http://openfarm.cc) is a free and open database and web application for farming and gardening knowledge. One might think of it as the Wikipedia for growing plants, though it functions more like a cooking recipes site. 
+
+The main content are Growing Guides: creative, crowd-sourced, single-author, structured documents that include all of the necessary information for a person or machine to grow a plant, i.e.: seed spacing and depth, watering regimen, recommended soil composition and companion plants, sun/shade requirements, etc. In this Freebase platform, gardeners can find answers to questions like &ldquo;How do I grow tomatoes?&rdquo; 
 
 ### Start by joining the discussion of existing Contributors
 
-To start the discussion, get involved, and meet OpenFarm core community of contributors, we strongly recommend joining [our Slack room](http://slack.openfarm.cc/)!. This is where you'll find the latest conversation about Openfarm and the core. 
+To start the discussion, get involved, and meet OpenFarm core community of contributors, we strongly recommend joining [our Slack room](http://slack.openfarm.cc/)! This is where you'll find the latest conversation about Openfarm and the most active contributors. 
 
-As an alternative, you can ask quick questions on [IRC on Freenode #openfarmcc](https://webchat.freenode.net/?channels=openfarmcc). The room might look empty but we have a bot piping messages back and forth with Slack.
+As a not-as-good-as Slack alternative, you can go to [IRC on Freenode #openfarmcc](https://webchat.freenode.net/?channels=openfarmcc) ; The room must look empty but we have a bot piping messages back and forth with Slack.
```

Comment:
Actually, we should just remove this text. The bot that pipes messages back and forth is broken.